composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "onelogin/php-saml",
     "description": "OneLogin PHP SAML Toolkit",
     "license": "MIT",
-    "version": "2.10.7",
+    "version": "2.10.8",
```

Comment:
We need to do this so that when we require it in https://github.com/intellihr/lapis the `aacotroneo/laravel-saml2` requirement can find a valid version to install. Ideally we could require `dev-master` for now.